src/components/RangeSelector.js

```diff
@@ -92,6 +93,47 @@ class RangeSelector extends React.Component {
     });
   }
 
+  _handleTrackMouseDown (e) {
```

Comment:
I played around with this method a bit and came up with a more condensed version of your changes that I think we should go with:
```javascript
_handleTrackMouseDown (e) {
  const clientX = e.touches ? e.touches[0].clientX : e.clientX;
  const newPixels = clientX - ReactDOM.findDOMNode(this.refs.rangeSelector).getBoundingClientRect().left;
  const updatedState = {
    trackClicked: true
  };
  const clickBelowLower = newPixels < this.state.lowerPixels;
  const clickAboveUpper = newPixels > this.state.upperPixels;
  const clickCloserToLower = newPixels > this.state.lowerPixels && newPixels < (this.state.lowerPixels + (this.state.upperPixels - this.state.lowerPixels) / 2);
  const clickCloserToUpper = newPixels < this.state.upperPixels && newPixels > (this.state.upperPixels - (this.state.upperPixels - this.state.lowerPixels) / 2);

  if (clickBelowLower || clickCloserToLower) {
    updatedState.dragging = 'Lower';
  }

  if (clickAboveUpper || clickCloserToUpper) {
    updatedState.dragging = 'Upper';
  }

  this.setState(updatedState);
}
```